paf-mvp-frontend/tests/test-cases/queue.test.ts

```diff
@@ -0,0 +1,58 @@
+import { TestWatcher } from 'jest';
+import { isExportDeclaration } from 'typescript';
+import { Command, CommandQueue, processCommands } from '../../src/utils/queue';
+
+describe('queue', () => {
+  let cmd1: jest.Mock;
+  let cmd2: jest.Mock;
+  let cmd3: jest.Mock;
+  const lib = {
+    queue: [] as CommandQueue,
+  };
+
+  beforeEach(() => {
+    cmd1 = jest.fn();
+    cmd2 = jest.fn();
+    cmd3 = jest.fn();
+    lib.queue = [];
+  });
+
+  test('processCommands undefined do not crash', () => {
+    lib.queue = processCommands(undefined);
```

Comment:
the affectation plays no role here, suggestion:
```typescript
processCommands(undefined);
// Assert no exception thrown
```

Reply:
still for the example of the usage